dagmc_stats/DagmcQuery.py

```diff
@@ -43,3 +54,17 @@ def get_tris(self):
             tris = self.dagmc_file._my_moab_core.get_entities_by_type(item, types.MBTRI)
             tris_lst.extend(tris)
         return tris
+
+    def calc_tris_per_vert(self, ignore_zero=True):
+        """
+        popoulate triangle per vertex
+        """
+        t_p_v_data = []
+        tri_dimension = 2
+        for vertex in self.dagmc_file.native_ranges[types.MBVERTEX]:
+            tpv_val = self.dagmc_file._my_moab_core.get_adjacencies(vertex, tri_dimension).size()
+            if ignore_zero and tpv_val == 0:
+                continue
+            t_p_v_data.append(tpv_val)
+        self._vert_data['t_p_v'] = np.array(t_p_v_data)
```

Comment:
let's actually call this `"Triangles per vertex"`

Reply:
also is this creating a new column different from the `tris_per_vert` column created in the init?

Reply:
and do we know if these are getting put into the data frame at the correct vertex entity handle?

Reply:
maybe a list of dictionaries: `t_p_v_data.append({'vert_eh': vertex, `tris_per_vert`: tpv_val})`